lib/commands/add.ts

```diff
@@ -47,7 +48,13 @@ command = {
 			config.project.projectType
 		) as ProjectLibrary;
 
-		if (!frameworkLibrary.hasTemplate(argv.template)) {
+		if (!argv.template && !argv.name) {
+			const prompts = new PromptSession(command.templateManager);
+			await prompts.chooseActionLoop(frameworkLibrary, config.project.theme);
+			return;
+		}
+
+		if (argv.template && argv.name && (!frameworkLibrary.hasTemplate(argv.template))) {
```

Comment:
The `argv.template && argv.name` check here should be either separate (to show help) or not here at all.
What happens if just name is passed? No feedback at all?
The following `hasTemplate` check kind of does the same too?